packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -15,7 +15,9 @@ Start by signing in to the Okta Developer Console:
 
 4. For allowed grant types, make sure that both **Authorization Code** and **Refresh Token** are selected. This enables PKCE flow for your application and the ability to refresh the access token when it expires without prompting the user to reauthenticate. [Refresh token rotation](/docs/guides/refresh-tokens/refresh-token-rotation) is automatically set as the default refresh token behavior.
 
-> **Note:** There is a default grace period for token rotation of 30 seconds. You can [change the value](/docs/guides/refresh-tokens/refresh-token-rotation/#enable-refresh-token-rotation) to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there is no grace period.
+    Refresh token rotation is an <ApiLifecycle access="ea"/> feature.
```

Comment:
I would not say "Refresh token rotation" here - for non-SPA, that is a true statement, but for SPA, even having Refresh tokens at all, with or without rotation, is new.  Suggest: "Refresh tokens for SPA"

Reply:
Also, in case you are unaware of how to turn on a self-service feature (I was), I'm told it is:  "enabled via Self-Service in the Admin Console under Settings -> Features"

Reply:
please see the screen shot and the content update.